godtools/ViewControllers/Platform/LanguageSettingsViewController.swift

```diff
@@ -15,30 +15,51 @@ protocol LanguageSettingsViewControllerDelegate {
 class LanguageSettingsViewController: BaseViewController {
     
     var delegate: LanguageSettingsViewControllerDelegate?
+    let languagesManager = LanguagesManager.shared
+    
+    @IBOutlet weak var primaryLanguageButton: BlueButton!
+    @IBOutlet weak var parallelLanguageButton: BlueButton!
     
     override var screenTitle: String {
         get {
             return "language_settings".localized
         }
     }
     
-    override func viewDidLoad() {
-        super.viewDidLoad()
+    override func viewWillAppear(_ animated: Bool) {
+        super.viewWillAppear(animated)
         
-        _ = DownloadedResourceManager.shared.loadFromRemote()
+        setupPrimaryLanguageButton()
+        setupParallelLanguageButton()
+    }
+    
+    private func setupPrimaryLanguageButton() {
+        let primaryLanguage = self.languagesManager.loadPrimaryLanguageFromDisk()
+        if primaryLanguage != nil {
+            primaryLanguageButton.setTitle(primaryLanguage!.localizedName, for: .normal)
```

Comment:
This is just coding style, but what do you think about storing the language name into a variable, so you would call the `setTitle` only once for the variable?